InternshipProject#2/Helpers/GetUserIdClaimValue.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace InternshipProject_2.Helpers;
+
+public class GetUserIdClaimValue
```

Comment:
You can merge all the user/token operations into one single helper.

Reply:
merged to one helper